src/components/project/project-member/project-member.service.ts

```diff
@@ -207,7 +208,10 @@ export class ProjectMemberService {
     const user = await forUser();
     const availableRoles = user.roles.value ?? [];
     const forbiddenRoles = difference(roles, availableRoles);
-    if (forbiddenRoles.length) {
+    if (
+      forbiddenRoles.length &&
+      !session.roles.includes('global:Administrator')
```

Comment:
Move this check to the if statement above so we don't call User.readOne when we don't need to.

Reply:
Nevermind. This isn't needed. See comment on UI PR.

There may need to be some changes to the role files to limit who can change User.roles.